i18n/states/PY.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Paraguay states
+ *
+ * @package WooCommerce/i18n
+ * @version 2.0.0
+ */
+
+global $states;
+
+defined( 'ABSPATH' ) || exit;
+
+$states['PY'] = array(
+    'Alto Paraguay' => __( 'Alto Paraguay', 'woocommerce' ),
```

Comment:
We should use ISO 3166 in here too, maybe get from https://en.wikipedia.org/wiki/ISO_3166-2:PY